docs/source/contributing/translation.rst

```diff
@@ -0,0 +1,51 @@
+====================
+Internationalization
+====================
+
+If you want to help with translating the docs:
+
+#. Make sure you have a `Transifex <https://www.transifex.com/>`_ account. Transifex is a tool for colaborative translation and allows Manim team to review and help to improve suggestions from the community. You will use Transifex to submit your translation suggestions.
+#. Open to the `Manim project page <https://www.transifex.com/manim-community/manim-community-documentation>`_ .
+#. Click on Join Team.
+#. Select the language you want to help with.
```

Comment:
```suggestion
#. Select the language(s) you want to help with.
```

Reply:
Well, you can only select one language in the combobox, so i think language without (s) is more accurate.

Reply:
I think "a" instead of "the" would be better in that case then.